.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: python
+
+python:
+   
+  - "3.7"
+
+services:
+  - postgresql
+
+env:
+  - DJANGO_VERSION=2.0.7
+
+before_script:
+  - psql -c 'create database test_db;' -U postgres
+ 
+install:
+  - pip install -r requirements.txt
+
+script:
+  - coverage run --source='.' manage.py test 
+
+after_success:
+  - coveralls
```

Comment:
Do we have coveralls installed?

Reply:
Yes, it is.

Reply:
@AGMETEOR I don't see it in requirements.txt 🤔

Reply:
The red arrow shows that you need to add a new line at the end of the file. Please add it.